apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/BranchController.java

```diff
@@ -0,0 +1,139 @@
+                                  @PathVariable String namespaceName) {
+
+        return namespaceBranchService.findBranch(appId, Env.valueOf(env.toUpperCase()), clusterName, namespaceName);
+    }
+
+    @RequestMapping(value = "/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/operator/{operator}/branches", method = RequestMethod.POST)
+    public NamespaceDTO createBranch(@PathVariable String appId,
+                                     @PathVariable String env,
+                                     @PathVariable String clusterName,
+                                     @PathVariable String namespaceName,
+                                     @PathVariable String operator,
+                                     HttpServletRequest request) {
+
+        return namespaceBranchService.createBranch(appId, Env.valueOf(env.toUpperCase()), clusterName, namespaceName, operator);
+    }
+
+    @RequestMapping(value = "/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/branches/{branchName}/operator/{operator}", method = RequestMethod.DELETE)
+    public void deleteBranch(@PathVariable String appId,
+                             @PathVariable String env,
+                             @PathVariable String clusterName,
+                             @PathVariable String namespaceName,
+                             @PathVariable String branchName,
+                             @PathVariable String operator) {
+/**
+ TBD
+        boolean canDelete = permissionValidator.hasReleaseNamespacePermission(appId, namespaceName) ||
+                (permissionValidator.hasModifyNamespacePermission(appId, namespaceName) &&
+                        releaseService.loadLatestRelease(appId, Env.valueOf(env.toUpperCase()), branchName, namespaceName) == null);
+
+
+        if (!canDelete) {
+            throw new AccessDeniedException("Forbidden operation. "
+                    + "Caused by: 1.you don't have release permission "
+                    + "or 2. you don't have modification permission "
+                    + "or 3. you have modification permission but branch has been released");
+        }
+*/
+        namespaceBranchService.deleteBranch(appId, Env.valueOf(env.toUpperCase()), clusterName, namespaceName, branchName, operator);
+
+    }
+
+
+
+    @PreAuthorize(value = "@consumerPermissionValidator.hasModifyNamespacePermission(#request, #appId, #namespaceName)")
+    @RequestMapping(value = "/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/branches/{branchName}/operator/{operator}/merge", method = RequestMethod.POST)
+    public ReleaseDTO merge(@PathVariable String appId, @PathVariable String env,
+                            @PathVariable String clusterName, @PathVariable String namespaceName,
+                            @PathVariable String branchName, @PathVariable String operator, @RequestParam(value = "deleteBranch", defaultValue = "true") boolean deleteBranch,
```

Comment:
normally, operator should be a RequestParam(query parameter) instead of a PathVariable